src/ClientHandle.cpp

```diff
@@ -1407,7 +1407,8 @@ void cClientHandle::HandleBlockDigFinished(int a_BlockX, int a_BlockY, int a_Blo
 	}
 	else
 	{
-		World->DigBlock(absPos);
+		// Using the chunk maps DigBlock function to prevent calling BreakBlock and OnPlayerBrokeBlock simultaneously
+		World->GetChunkMap()->DigBlock(absPos);
```

Comment:
This is actually the main change. Just skipping the call to World.DigBlock() which calls the OnBreak and then the Chunkmap.DigBlock()

Reply:
migt wanna revert this ...